openssl/src/x509/mod.rs

```diff
@@ -467,6 +476,34 @@ impl X509Ref {
         }
     }
 
+    /// Returns this certificate's key usage extension value, if it exists.
+    #[corresponds(X509_get_ext_d2i)]
+    pub fn key_usage(&self) -> Option<&Asn1BitStringRef> {
+        unsafe {
+            let ptr = ffi::X509_get_ext_d2i(
```

Comment:
For consistency with the OpenSSL API we're now implementing these APIs with a single `extension` function, like this: https://docs.rs/openssl/latest/openssl/x509/struct.X509Revoked.html#method.extension